antarest/study/service.py

```diff
@@ -1856,6 +1855,25 @@ def create_link(
         )
         return new_link
 
+    def update_link(
+        self,
+        uuid: str,
+        link_update_dto: LinkInfoDTOType,
+        params: RequestParameters,
+    ) -> LinkInfoDTOType:
+        study = self.get_study(uuid)
+        assert_permission(params.user, study, StudyPermissionType.WRITE)
+        self._assert_study_unarchived(study)
+        updated_link = self.links.update_link(study, link_update_dto)
+        self.event_bus.push(
```

Comment:
We usually don't push this inside the eventbus but perhaps it's a good idea. I think it can flood the bus but not sure

Reply:
I saw this on area updating and other services but I can remove it